Gaming The Syllabus/Assets/Scripts/Combat/UI/CombatStarter.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CombatStarter : MonoBehaviour
+{
+    //[SerializeField] private GameObject combatUIGameObject;
+    [SerializeField] private List<string> enemyFilePaths;
```

Comment:
Consider adding a tooltip here to let people using the inspector know that this filepath shouldn't include "Assets/" at the front